instrumentation/opentelemetry-instrumentation-asgi/src/opentelemetry/instrumentation/asgi/__init__.py

```diff
@@ -372,24 +426,30 @@ def get_host_port_url_tuple(scope):
     return server_host, port, http_url
 
 
-def set_status_code(span, status_code):
+def set_status_code(
+    span,
+    status_code,
+    metric_attributes=None,
+    sem_conv_opt_in_mode=_HTTPStabilityMode.DEFAULT,
+):
     """Adds HTTP response attributes to span using the status_code argument."""
     if not span.is_recording():
         return
+    status_code_str = repr(status_code)
```

Comment:
Just curious, why not use `str` here?

Reply:
No real reason. Switched to using `str`.